compose/ui/ui-test/src/desktopMain/kotlin/androidx/compose/ui/test/Mouse.desktop.kt

```diff
@@ -28,16 +29,16 @@ actual value class MouseButton(val buttonId: Int) {
         /**
          * The left mouse button
          */
-        actual val Primary = MouseButton(1)
+        actual val Primary = MouseButton(0)
```

Comment:
Yes, it was copy-pasted from Android, where these id's are represented as bit masks (MotionEvent.BUTTON_PRIMARY, ...). Because it was initially isn't implemented the right way, it is experimental, and it isn't worked anyway, we can make changes to these ids. For simplicity of conversions and for the future migration to PointerButton, it is better to treat these ids as indices.